source/fundamentals/enterprise-auth.txt

```diff
@@ -280,3 +287,150 @@ mechanism:
       method. The code to instantiate a ``MongoClient`` resembles the following:
 
       .. include:: /includes/fundamentals/code-snippets/auth-credentials-ldap.rst
+
+.. _mongodb-oidc:
+
+``MONGODB-OIDC``
```

Comment:
S: since the other authentication mechanisms aren't monospaced, I think this header format should align with those
```suggestion
MONGODB-OIDC
```